common/src/main/java/org/apache/uniffle/common/config/RssClientConf.java

```diff
@@ -142,4 +142,12 @@ public class RssClientConf {
           .enumType(ClientType.class)
           .defaultValue(ClientType.GRPC)
           .withDescription("Supports GRPC, GRPC_NETTY");
+
+  public static final ConfigOption<Boolean> RSS_CLIENT_REMOTE_STORAGE_USE_LOCAL_CONF_ENABLED =
+      ConfigOptions.key("rss.client.remote.storage.use-local-conf.enabled")
```

Comment:
Could we use 4 levels + camel naming style? This is the style of Spark. This will avoid too many levels.
For example:
`rss.client.remote.storage.useLocalConfAsDefault`.

Reply:
Got it.